TitleBarCustomization/explainer.md

```diff
@@ -86,18 +86,18 @@ The solution proposed in this explainer is in multiple parts
 3. New CSS environment variables to define the left and right insets from the edges of the window: `unsafe-area-top-inset-left` and `unsafe-area-top-inset-right`
 4. A standards-based way for developers to define system drag regions on their content
 
-### Overlaying Window Controls
+### Overlaying Window Controls on a Minimal-frame Window
```

Comment:
We should probably avoid "minimal" due to conflict with "Minimal-UI". Maybe a "frameless" window?

Reply:
Agree